### ISSUE TITLE
False positive: unused class property if it used in trait only

### ISSUE THREAD

**Issue by Koc** [kalessil/phpinspectionsea#1784]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended)  |
| Language level | 8.0                     |

Current behaviour
--
![image](https://user-images.githubusercontent.com/191082/144624813-d42bdec2-fdab-4bbf-832b-e17b9fef3d3e.png)

Expected behaviour
--
Actually `reader` property used in `TranslationTrait`.

Environment details
---

```
PhpStorm 2021.3
Build #PS-213.5744.279, built on December 2, 2021
Licensed to Konstantin Myakshin
Subscription is active until April 5, 2022.
Runtime version: 11.0.13+7-b1751.19 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.11.0-41-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 1500M
Cores: 8
Registry:
    run.processes.with.pty=TRUE
    documentation.show.toolbar=true

Non-Bundled Plugins:
    socrates.tabshifter (0.30)
    org.wavescale.sourcesync (1.9)
    org.intellij.RegexpTester (1.0.8)
    org.github.erikzielke.gotoproject (1.2)
    name.kropp.intellij.makefile (213.5744.224)
    com.jetbrains.annotations.preloader (1.0.4)
    com.intellij.ideolog (203.0.27.0)
    org.intellij.plugins.postcss (213.5744.190)
    TabSwitch (3.0.8)
    String Manipulation (8.22.203.000.1)
    Scratch (1.10)
    Identifier Highlighter Reloaded (1.02.1)
    uk.co.ben-gibson.remote.repository.mapper (3.3.6)
    de.espend.idea.php.phpunit (6.2)
    com.udari (0.2.4)
    com.ruslanpolutsygan.adderremover (0.3.6)
    com.kalessil.phpStorm.phpInspectionsEA (4.0.6.5)
    de.espend.idea.php.toolbox (6.0.0)
    de.espend.idea.php.annotation (8.0.0)
    fr.adrienbrault.idea.symfony2plugin (0.23.213)
    com.github.woru.options-completion-phpstorm-plugin (0.0.7)
    com.dryabov.phpStorm.phpregexp (0.9.7)
    NEON support (0.5.1)
    tv.twelvetone.intellij.plugins.intellivue (1.1.9)
    de.timo_reymann.intellij-mjml-support (1.20.2)
    com.magento.idea.magento2plugin (4.2.0)
    com.jetbrains.lang.ejs (213.5744.190)
    lv.midiana.misc.phpstorm-plugins.deep-keys (2021.07.18.002)
    com.ppolivka.plugin.gitcleaner (1.1.4)
    FrameSwitcher (3.29.0-212)
    com.intellij.kubernetes (213.5744.122)
    ru.adelf.idea.dotenv (2021.3.0.213)
    CMD Support (1.0.5)
    lermitage.intellij.extra.icons (1.67.0.203)

Current Desktop: Pantheon
```

**Comment by ea-inspections-team** [comment]
Thanks for reporting @Koc! Fixed!
